bots/reads.ts

```diff
@@ -392,7 +420,14 @@ const handleUpdateUrl = async (url: string, ctx: ReadsContext, config: ReadsConf
 
   try {
     metadata = await fetchUrlMetadata(cleanUrl, config);
+    await readLinkRecentlyAdded(cleanUrl, config);
   } catch (e) {
+    if (e.message === ERROR_DUPLICATE_READ) {
+      await ctx.reply('Oops, this url was recently added already');
+      await handleNew(ctx);
```

Comment:
in the above handling of a duplicate, we're doing a `resetState(ctx)`. should it be the same here?

Reply:
ah, prob should be consistent - this was consistent with a few lines down where we re-prompt on a new line to fetch the url - might make more sense to call handleNew() in other catch